results/broadwell/sbatch_furiosa_fftw_broadwell.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#SBATCH -J gearshifftFFTW
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --cpus-per-task=40
+#SBATCH --time=110:00:00
+#SBATCH --mem=128991M
+#SBATCH --partition=gpu
+#SBATCH --exclusive
+#SBATCH --array 0-5,7-9
+#SBATCH -o slurmcpu_array-%A_%a.out
+#SBATCH -e slurmcpu_array-%A_%a.err
+
+k=$SLURM_ARRAY_TASK_ID
+
+CURDIR=/home/steinbac/development/gearshifft/
+REL=$CURDIR/build-brdw
+RESULTS=$CURDIR/results/broadwell/fftw3.3.6pl1
+mkdir -p ${RESULTS}
+
+FEXTENTS1D=$CURDIR/config/extents_1d_publication.conf
+FEXTENTS1DFFTW=$CURDIR/config/extents_1d_fftw.conf  # excluded a few very big ones
+FEXTENTS2D=$CURDIR/config/extents_2d_publication.conf
+FEXTENTS3D=$CURDIR/config/extents_3d_publication.conf
+FEXTENTS=$CURDIR/config/extents_all_publication.conf
+
+module purge
+module load fftw/3.3.6-pl1-brdw cuda/8.0.61 boost/1.63.0
+module unload gcc
+module load gcc/5.3.0
+
+if [ $k -eq 0 ]; then
+#   srun $REL/gearshifft_fftw -f $FEXTENTS1D -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.1d.csv --rigor estimate # got killed by OS (OOM error)
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS1DFFTW -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.1d.csv --rigor estimate
+
+elif [ $k -eq 1 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS2D -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.2d.csv --rigor estimate
+
+elif [ $k -eq 2 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS3D -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.3d.csv --rigor estimate
+
+elif [ $k -eq 3 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS1DFFTW -o $RESULTS/fftw_wisdom_gcc5.3.0_RHEL7.2.1d.csv --rigor wisdom --wisdom_sp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt --wisdom_dp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt
```

Comment:
`--wisdom_dp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt`
should be
`--wisdom_dp $CURDIR/results/broadwell/fftw_wisdom_3.3.6-pl1_bdw.txt`
same applies for 2d and 3d below.